docs/x509/reference.rst

```diff
@@ -1695,6 +1695,27 @@ X.509 Extensions
 
         :type: int
 
+.. class:: CRLNumber(crl_number)
+
+    .. versionadded:: 1.2
+
+    The CRL number is a non-critical CRL extension that conveys a
```

Comment:
That comes from the RFC, but it means it should not be marked as critical. We could just strip it though.